src/leveled_imanifest.erl

```diff
@@ -28,9 +28,7 @@
 -type manifest() :: list({integer(), list()}).
 %% The manifest is divided into blocks by sequence number, with each block
 %% being a list of manifest entries for that SQN range.
--type manifest_entry() :: {integer(), string(), pid()|string(), any()}.
-%% The Entry should have a pid() as the third element, but a string() may be
-%% used in unit tests
+-type manifest_entry() :: {integer(), string(), pid(), any()}.
```

Comment:
Penciller manifest manifest_entry record now moved from global include, and put within leveled_pmanifest module - so that all references to it are now module specific.